test/stdlib/Result.swift

```diff
@@ -0,0 +1,151 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import Swift
+
+let ResultTests = TestSuite("Result")
+
+fileprivate enum Err: Error, Equatable {
+  case err
+  case derr
+}
+
+fileprivate let string = "string"
+
+ResultTests.test("Construction") {
+  let result1: Result<String, String> = .success(string)
+  let result2: Result<String, String> = .failure(string)
+  let string1: String? = {
+    switch result1 {
+      case let .success(string): return string
```

Comment:
Wrap to next line here and in the next line.
```suggestion
      case let .success(string): 
        return string
```